test/fixtures/autofix/jQuerySap.js

```diff
@@ -0,0 +1,237 @@
+sap.ui.define(["sap/base/strings/NormalizePolyfill"], function (NormalizePolyfill) {
+	// https://github.com/SAP/ui5-linter/issues/520
+	jQuery.sap.assert(false, "That's an assert");
+
+	// https://github.com/SAP/ui5-linter/issues/521
+	const bundle = jQuery.sap.resources({
+		url: "resources/i18n.properties"
+	});
+	// Respect the context
+	sap.ui.require(["sap/base/i18n/ResourceBundle"], function (innerScopeResourceBundle) {
+		const bundle = jQuery.sap.resources({
+			url: "resources/i18n.properties"
+		});
+	});
+
+	// https://github.com/SAP/ui5-linter/issues/522
+	const logObject = jQuery.sap.log;
+	const myLogger = jQuery.sap.log.getLogger();
+	const myLog = jQuery.sap.log.getLog();
+	const myLog2 = jQuery.sap.log.getLogEntries();
+
+	const oLogListener = {};
+	oLogListener.onLogEntry = function(oLog) {
+		//
+	};
+	jQuery.sap.log.addLogListener(oLogListener);
+	jQuery.sap.log.removeLogListener(oLogListener);
+
+	const level = jQuery.sap.log.getLevel();
+	const logLevel = jQuery.sap.log.Level;
+	const isLoggable = jQuery.sap.log.isLoggable(logLevel.DEBUG);
+
+	jQuery.sap.log.logSupportInfo(true);
+	jQuery.sap.log.debug("This is a debug log message");
+	jQuery.sap.log.error("This is a error log message");
+	jQuery.sap.log.fatal("This is a fatal log message");
+	jQuery.sap.log.info("This is a info log message");
+	jQuery.sap.log.trace("This is a trace log message");
+	jQuery.sap.log.warning("This is a warning log message");
+
+	// Do not migrate those!
```

Comment:
The "MethodChaining" QUnit test in openui5 fails, so I think there's something missing about the return value check.

Please also add a note here why those cases should not be migrated.

Reply:
I'm updating the comment